includes/Features/Abilities_Explorer/assets/js/abilities-explorer.js

```diff
@@ -0,0 +1,388 @@
+/**
```

Comment:
General comment: there are a bunch of coding standards errors in the JS file, suggest running the linter to see the output
```
npm run lint:js
```